unit/solvers/smt2_incremental/smt_to_smt2_string.cpp

```diff
@@ -21,6 +21,9 @@ TEST_CASE("Test smt_sortt to string conversion", "[core][smt2_incremental]")
 {
   CHECK(smt_to_smt2_string(smt_bool_sortt{}) == "Bool");
   CHECK(smt_to_smt2_string(smt_bit_vector_sortt{16}) == "(_ BitVec 16)");
+  CHECK(
+    smt_to_smt2_string(smt_array_sortt{smt_bool_sortt{}, smt_bool_sortt{}}) ==
+    "(Array Bool Bool)");
```

Comment:
Good to check more than one single sort here, recursive and bitvector sorts would be good to check.